examples/cluster/cluster.json

```diff
@@ -0,0 +1,44 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "This template creates a cluster on the MongoDB Atlas API, this will be billed to your Atlas account.",
+    "Parameters": {
+        "ProjectName": {
+            "Type": "String"
+        }
+    },
+    "Mappings": {},
+    "Resources": {
+        "Cluster": {
+            "Type": "MongoDB::Atlas::Cluster",
+            "Properties": {
+                "ProjectId": "5cf5a45a9ccf6400e60981b6",
```

Comment:
Can you add an example for `Project` and pass it with a Ref instead of hardcoded?